samples/snippets/src/test/java/SnippetsIT.java

```diff
@@ -134,7 +116,7 @@ public void testIdTokenFromMetadataServer() throws GeneralSecurityException, IOE
   @Test
   public void testAuthenticateImplicitWithAdc() throws IOException {
     AuthenticateImplicitWithAdc.authenticateImplicitWithAdc(PROJECT_ID);
-    assertThat(stdOut.toString()).contains("Listed all storage buckets.");
```

Comment:
See AuthenticateImplicitWithAdc(). It believe it prints out "Listed all storage buckets."